NEWS

```diff
@@ -1,3 +1,14 @@
+0.2.0 - 2019-11-12
+
+  - Remove dependency on BasePlugin (#50, #51)
+  - Replace `kong.request` span name with `method url` (#52)
+  - Rename `component` tag to `lc` (#52)
+  - Restructure of Kong generated spans (#52)
+  - Change the name of spans for http traffic to `GET`
```

Comment:
remove "instead of `GET http://foo.bar/baz` in order to avoid excessive cardinality". That naming convention (with the url) never made it to an official version of this plugin, it was only briefly on master for 1 or 2 days.